src/index.html

```diff
@@ -17,6 +17,87 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="main-header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Company Logo"
+        />
+      </a>
+      <nav>
+        <ul class="nav-menu">
+          <li>
+            <a
+              href="#"
+              class="nav-link is-active"
+            >
+              APPLE
+            </a>
```

Comment:
The `href` attribute in the `<a>` tag should not contain spaces. Use a meaningful reference or `#` if the link is a placeholder.